### PR TITLE
Use replies instead of a private channel for /look_around

### PR DESCRIPTION
**Pull request by nathanl**
No need for an extra connection just to send private messages to a
connected user; using replies accomplishes the same thing more simply,
as Jay had pointed out in a comment

### CODE REVIEW

File: web/static/js/socket.js [hunk 1]
```diff
@@ -69,19 +71,22 @@ $chatInput.on("keypress", event => {
...
-  $messagesContainer.prepend(`<li>[${Date()}] ${payload.from}: ${payload.body}</li>`)
+// Really broad event - captures more than we want
+channel.on("phx_reply", payload => {
+  let response = payload.response
+  // Hacky way to narrow down to the replies we're interested in
+  if(payload.status === "ok" && response && response.kind === "private" && Array.isArray(response.messages)) {
+    response.messages.forEach(body => {
+      displayMessage({from: response.from, body: body})
+    })
+  }
```
Comment (iamvery) [inline web/static/js/socket.js]: I wonder if this is the most correct want to implement replies? I noticed that you can also:

``` javascript
channel.push("new_msg", body: "foo")
  .receive("ok", (resp) => console.log(resp))
```

Comment (iamvery) [inline web/static/js/socket.js]: Btw, I only know this because I dug into the phoenix JS. I haven't found any good documentation for that online...

Comment (nathanl) [inline web/static/js/socket.js]: Ha - yeah, I just learned that from asking on the mailing list. Updated the code accordingly.

**Comment by iamvery** [comment]
You could remove the "route" from the UserSocket too, ya?

**Comment by nathanl** [comment]
@jay Good point - done

**Pull request merged**